internal/config/testdata/config.yaml

```diff
@@ -1,5 +1,7 @@
 healthProbeBindAddress: :8081
 webhookPort: 9443
+job:
+  gcDelay: 1h
```

Comment:
Are we really waiting 1h for test to pass?

Reply:
This is a test config file for unit tests.